extensions/http/pom.xml

```diff
@@ -64,7 +64,6 @@
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-sockets</artifactId>
             <version>${project.version}</version>
-            <scope>provided</scope>
```

Comment:
Yes. We have fix classloading so extensions can depend on plugins with compile scope